lib/remote_helper.py

```diff
@@ -989,7 +989,7 @@ def copy_app_to_host(cls, app_location, keyname, is_verbose):
     rand = str(uuid.uuid4()).replace('-', '')[:8]
     local_tarred_app = "{0}/appscale-app-{1}-{2}.tar.gz".format(tempfile.gettempdir(),
       app_id, rand)
-    LocalState.shell("cd '{0}' && tar -czhf {1} --exclude='*.pyc' *".format(
+    LocalState.shell("cd '{0}' && COPYFILE_DISABLE=1 tar -czhf {1} --exclude='*.pyc' *".format(
```

Comment:
Funny you should mention Draco; my favorite legislator. 
Joking aside, it will only omit Apple metadata files used for HFS+.